lib/bolt/cli.rb

```diff
@@ -363,7 +363,11 @@ def execute(options)
             when 'command'
               executor.run_command(nodes, options[:object])
             when 'script'
-              executor.run_script(nodes, options[:object], options[:leftovers])
+              script = options[:object]
+              unless file_exist?(script)
```

Comment:
We should probably verify that it's readable too

Reply:
Updated, also check that it's a file, otherwise upload prints an undecipherable error message.